docs/usage.md

```diff
@@ -288,16 +288,23 @@ spec:
 If you set `replicas` to more than 1, normally you should not set `sessionAffinity` to `None`.
 This is because session affinity is mandatory to keep stateful TCP connections.
 
-One problem with session affinity in Kubernetes is that it limits the maximum
-duration of a session.  By default, [it is 3 hours](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#clientipconfig-v1-core).
+You can extend the timeout duration for idle connections with 
+You may need to extend the timeout setting for idle connections with `spec.sessionAffinityConfig` as follows:
 
-If `kube-proxy` is running in IPVS mode, you can specify `--ipvs-scheduler=sh` to allow setting `sessionAffinity` to `None`.
-`sh` determines the destination server based on the client IP.
-
-For more information, read:
+```yaml
+apiVersion: coil.cybozu.com/v2
+kind: Egress
+metadata:
+  namespace: other-network
+  name: egress
+spec:
+  # snip
+  sessionAffinityConfig:
+    clientIP:
+      timeoutSeconds: 43200
+```
 
-- https://www.programmersought.com/article/28252846600/
-- https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/#options
+The default timeout seconds is 10800 (= 3 hours).
```

Comment:
A link to the k8s API reference is removed. Is it OK?
> https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#clientipconfig-v1-core

Reply:
I think there is enough information for users.